src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,61 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class WinningLotto {
+
+    private final Lotto lotto;
+    private final List<Integer> lastWinningNumbers;
+    private final Map<Integer, Integer> winningCounts;
+    private double rateOfReturn;
+    private static final int THREE_MATCHES = 5000;
+    private static final int FOUR_MATCHES = 50000;
+    private static final int FIVE_MATCHES = 1500000;
+    private static final int SIX_MATCHES = 2000000000;
+
+    public WinningLotto(Lotto lotto, List<Integer> lastWinningNumbers){
+        this.lotto = lotto;
+        this.lastWinningNumbers = lastWinningNumbers;
+        this.winningCounts = new HashMap<>();
+    }
+
+    public Map<String, Object> winningResult(){
+
+        matchingWinningNumberCount();
+        calculateRateOfReturn();
+
+        Map<String, Object> winningResult = new HashMap<>();
+        winningResult.put("winningCounts", winningCounts);
+        winningResult.put("rateOfReturn", rateOfReturn);
+        winningResult.put("prizeMoney", Map.of(
+                3, THREE_MATCHES,
+                4, FOUR_MATCHES,
+                5, FIVE_MATCHES,
+                6, SIX_MATCHES
+        ));
+
+        return winningResult;
+    }
+
+    private void matchingWinningNumberCount(){
+        for(LottoNumbers lottoNumbers : lotto.lotto()){
```

Comment:
```suggestion
        LottoResult result = lotto.macth(winningLotto);
```
lotto에서 값을 꺼내기 보다 Lotto에 메시지를 보내 Lotto가 실행 결과를 반환하도록 구현하는 것은 어떨까?